toonz/sources/translations/german/tnztools.ts

```diff
@@ -0,0 +1,1005 @@
+        <translation>Das aktuelle Werkzeug kann nicht zum Bearbeiten eines Bewegungspfads angewendet werden.</translation>
+    </message>
+    <message>
+        <source>The current level is not editable.</source>
+        <translation>Die aktuelle Ebene ist nicht bearbeitbar.</translation>
+    </message>
+    <message>
+        <source>The current tool cannot be used on a Vector Level.</source>
+        <translation>Das aktuelle Werkzeug kann nicht auf einer Vector-Ebene angewendet werden.</translation>
+    </message>
+    <message>
+        <source>The current tool cannot be used on a Toonz Level.</source>
+        <translation>Das aktuelle Werkzeug kann nicht auf einer Toonz-Ebene angewendet werden.</translation>
+    </message>
+    <message>
+        <source>The current tool cannot be used on a Raster Level.</source>
+        <translation>Das aktuelle Werkzeug kann nicht auf einer Raster-Ebene angewendet werden.</translation>
+    </message>
+    <message>
+        <source>The current tool cannot be used on a Mesh Level.</source>
+        <translation>Das aktuelle Werkzeug kann nicht auf einer Mesh-Ebene angewendet werden.</translation>
+    </message>
+    <message>
+        <source>The current tool cannot be used on a mesh-deformed level</source>
+        <translation>Das aktuelle Werkzeug kann nicht auf einer Mesh-verformten-Ebene angewendet werden.</translation>
+    </message>
+    <message>
+        <source>The current frame is locked: any editing is forbidden.</source>
+        <translation>Der aktuelle Frame ist gesperrt: jegliches Bearbeiten ist verboten.</translation>
+    </message>
+    <message>
+        <source>Ok</source>
+        <translation>Ok</translation>
+    </message>
+    <message>
+        <source>Cancel</source>
+        <translation>Abbrechen</translation>
+    </message>
+    <message>
+        <source>Paste</source>
+        <translation>Einfügen</translation>
+    </message>
+    <message>
+        <source>Move Center</source>
+        <translation>Mittlere Position bewegen</translation>
+    </message>
+    <message>
+        <source>RGB Picker (R%1, G%2, B%3)</source>
+        <translation>RGB Wähler (R%1, G%2, B%3)</translation>
+    </message>
+    <message>
+        <source>Group</source>
+        <translation>Gruppieren</translation>
+    </message>
+    <message>
+        <source>Ungroup</source>
+        <translation>Entgruppieren</translation>
+    </message>
+    <message>
+        <source>Move Group</source>
+        <translation>Gruppe bewegen</translation>
+    </message>
+    <message>
+        <source>Modify Fx Gadget  </source>
+        <translation>FX Gadget modifizieren</translation>
+    </message>
+    <message>
+        <source>%1   Level : %2  Frame : %3</source>
+        <translation>%1   Ebene : %2  Frame : %3</translation>
+    </message>
+    <message>
+        <source>Modify Stroke Tool</source>
+        <translation>Streich-Werkzeug modifizieren</translation>
```

Comment:
I believe Streichen is a mistranslation here. I think stroke refers to the style of lines. In Inkscape it is translated with "Kontur". "Umrisslinie" is also a good, but unwieldy word.